frontend/src/components/dashboard/ProfileBanners.module.scss

```diff
@@ -5,3 +5,7 @@
   flex-direction: column;
   gap: $spacing-xl;
 }
+
+.bundle-logo {
+  min-width: 120px;
+}
```

Comment:
I don't think this is needed; the width is constrained by this line: https://github.com/mozilla/fx-private-relay/blob/db6098a8ff27f007bf402d25901f6a84797cf8f3/frontend/src/components/Banner.module.scss#L100

`git blame` indicates that that line was added in 5f5c4c4cc20f2e25127e719979e9badd86f0ba42, for a banner (`<NoPremiumBanner>`) that is no longer being shown. So instead of adding more CSS that reverts some other CSS, it's probably better to just delete the line linked above :)

Reply:
@Vinnl Hmm, from my experience that line actually affects all of the banners with illustrations in them, not just `<NoPremiumBanner>`

Reply:
Oh hmm, well, blame me then - would've been better to give the icons an explicit width I think. Ah well :P